src/main/java/net/sf/jabref/gui/groups/GroupSelector.java

```diff
@@ -250,7 +250,6 @@ public void stateChanged(ChangeEvent e) {
                 groupsRoot.addNewGroup(newGroup, panel.getUndoManager());
                 panel.markBaseChanged();
                 frame.output(Localization.lang("Created group \"%0\".", newGroup.getName()));
-                panel.getBibDatabaseContext().getMetaData().postChange();
```

Comment:
The metadata class now directly subscribes to changes in the group tree and relays these change events https://github.com/JabRef/jabref/pull/2112/files#diff-20cd356a80ee8ffb10c848005c89b949R74